tensorflow/contrib/periodic_resample/BUILD

```diff
@@ -69,6 +70,17 @@ py_library(
     ],
 )
 
+py_test(
```

Comment:
Not sure whether this change is correct. I have added it to run this test by  "bazel test " on Linux. If this is mistake and it is possible to  run test without this, please, tell me about proper way doing this.